### PR TITLE
Make the 'options' argument to generate_barcode optional

### PR DESCRIPTION
**Pull request by dlenski**
(split from #90)

### CODE REVIEW

**Comment by adamchainz** [comment]
👌

Just add note in HISTORY

**Comment by dlenski** [comment]
Just this…?

```rst
-------
History
-------

* Make options optional
```

File: HISTORY.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ History
 -------
 
 .. Insert new release notes below this line
+* Make the ``options`` argument to ``generate_barcode`` optional
```
Comment (adamchainz) [inline HISTORY.rst]: tests failing because reStructuredText wants a newline above this line

Comment (dlenski) [reply HISTORY.rst]: Right. I wasn't sure whether you wanted the "Insert new release notes" line kept there or not. Will fix in a sec.

**Pull request merged**